onadata/apps/api/permissions.py

```diff
@@ -313,3 +313,14 @@ def has_object_permission(self, request, view, obj):
             return super(OrganizationProfilePermissions,
                          self).has_object_permission(
                 request=request, view=view, obj=obj)
+
+
+class OpenDataViewSetPermissions(IsAuthenticated):
```

Comment:
After the comment below, this should probably be extending `ViewDjangoObjectPermissions` or `DjangoObjectPermissionsAllowAnon`.